lanelet2_io/include/lanelet2_io/io_handlers/OsmHandler.h

```diff
@@ -12,9 +12,11 @@ class OsmWriter : public Writer {
  public:
   using Writer::Writer;
 
-  void write(const std::string& filename, const LaneletMap& laneletMap, ErrorMessages& errors) const override;
+  void write(const std::string& filename, const LaneletMap& laneletMap, ErrorMessages& errors,
```

Comment:
I just realized we have an example for the new parameters, but we are not *documenting* them anywhere. Here would be a good place, probably.

Reply:
done